src/models/MatrixAction.ts

```diff
@@ -179,7 +179,14 @@ export class MatrixAction {
                     fileSize = " (" + Math.round(event.content.info.size / 1024) + "KB)";
```

Comment:
```suggestion
                    fileSize = " (" + Math.round(event.content.info.size / 1024) + "KiB)";
```
Divided by 1024, so it's [KiB](https://en.wikipedia.org/wiki/Kibibyte).

You didn't change this.. so feel free to opt for "Christian, please file this as a separate issue".

Reply:
Nah, this is good.